.github/workflows/automerge-pyup.yml

```
@@ -0,0 +1,37 @@
+name: "pyup Automerge"
+
+on:
+  pull_request:
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    
+    if: github.actor == 'pyup-bot'
+    steps:
+      - name: 'Wait for status checks'
+        id: waitforstatuschecks
+        uses: "WyriHaximus/github-action-wait-for-status@master"
+        with:
+          ignoreActions: automerge,WIP
+          checkInterval: 300
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
```

I usually don't see the single quotes around this secret, is there a specific reason for them? Probably doesn't matter at all

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

I don't either, but the YAML syntax kept failing on me for some reason, so I just left this as is, so I just copied this portion from their README. Never tested without it, but I think it's also fine. Will take a look in the test repo.

---

I see 😄  YAML can be confusing sometimes, yes. Not very important here, either way works I guess.

---

It works without the quotes, so idk what happened but it's working now :P